build.sbt

```diff
@@ -15,67 +15,55 @@
  */
 
 import Dependencies._
-import SparkVersionAxis._
 import JacocoSetup._
 import sbt.Keys.name
 
-
-ThisBuild / organization := "za.co.absa"
+ThisBuild / organization := "za.co.absa.atum-service"
```

Comment:
Why? The organization is not atum-service. That's project name. But I can see in https://www.scala-sbt.org/1.x/docs/Howto-Project-Metadata.html#Set+the+project+organization, that:

`By convention, this is a reverse domain name that you own, typically one specific to your project. It is used as a namespace for projects.`

So probably okay like this...although we have organization=za.co.absa in many repos at the moment.

But in order for `sbt-ci-release` to work properly, we have to add also `sonatypeProfileName` - if not specified explicitly, `organization` will be used - but in this case this organization `"za.co.absa.atum-service"` doesn't exist so that release would fail. I just noticed in balta. 

So either 

ThisBuild / organization := "za.co.absa.atum-service"
sonatypeProfileName := "za.co.absa"

or

ThisBuild / name := "atum-service"
ThisBuild / organization := "za.co.absa"

(I vote for the later one, as it makes more sense to me)

Reply:
Also, in sbt manual, `organization` is:

`organization := "org.example"`

and 

A full/formal name can be defined in the organizationName setting. This is used in the generated pom.xml. If the organization has a web site, it may be set in the organizationHomepage setting. For example:
 
```
organizationName := "Example, Inc."
organizationHomepage := Some(url("http://example.org"))
https://www.scala-sbt.org/1.x/docs/Howto-Project-Metadata.html#Set+the+project+organization
```

this suggests that the organization has a meaning of a 'company', not of a 'project structure organizing'. :D